VAGRANT.md

```diff
@@ -45,11 +45,11 @@ is.
 
   ```
   # inside the virtual machine:
-  cd Nominatim
-    wget --no-verbose --output-document=data/monaco.osm.pbf http://download.geofabrik.de/europe/monaco-latest.osm.pbf
-    ./utils/setup.php --osm-file data/monaco.osm.pbf --osm2pgsql-cache 1000 --all 2>&1 | tee monaco.$$.log
-    ./utils/specialphrases.php --countries > data/specialphrases_countries.sql
-    psql -d nominatim -f data/specialphrases_countries.sql
+  cd Nominatim/build-vagrant/
```

Comment:
I've moved that directory. The vagrant scripts now have their build dir in /home/vagrant/build.